MonoGame.Framework/Content/ContentManager.cs

```diff
@@ -506,6 +540,27 @@ private ContentReader GetContentReaderFromXnb(string originalAssetName, Stream s
 
             return reader;
         }
+    
+        object LoadTexture2DHeuristically(string assetName)
```

Comment:
This will either return `Texture2D` or null correct? If so, could this return `Texture2D` instead?